bin/mklittlefs.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import getopt
+import sys
+import os
+from littlefs import LittleFS
```

Comment:
this requires littlefs-python to be installed via PIP. The littlefs implementation depends on Cython, so that needs to be installed as well.

Reply:
good call out.

Reply:
will our CI need to be updated for this?

Reply:
yes, at least mine had to. i am spinning regular private docker images with all the tools in them, so i won't download the same stuff all over again. with github ci it might be as easy as adding 'littlefs-python' to the platformio setup. Cython should be an operating system package.